solutions/standard/main.tf

```
@@ -434,10 +434,12 @@ module "secrets_manager_service_credentials" {
 ########################################################################################################################
 
 locals {
-  code_engine_project_id   = var.existing_code_engine_project_id != null ? var.existing_code_engine_project_id : null
-  code_engine_project_name = local.code_engine_project_id != null ? null : (var.prefix != null && var.prefix != "") ? "${var.prefix}-code-engine-kibana-project" : "ce-kibana-project"
-  code_engine_app_name     = (var.prefix != null && var.prefix != "") ? "${var.prefix}-kibana-app" : "ce-kibana-app"
-  kibana_version           = var.enable_kibana_dashboard ? jsondecode(data.http.es_metadata[0].response_body).version.number : null
+  default_code_engine_project_name = (var.kibana_code_engine_new_project_name != null && var.kibana_code_engine_new_project_name != "") ? var.kibana_code_engine_new_project_name : "ce-kibana-project"
+  default_code_engine_app_name     = (var.kibana_code_engine_new_app_name != null && var.kibana_code_engine_new_app_name != "") ? var.kibana_code_engine_new_app_name : "ce-kibana-app"
+  code_engine_project_id           = var.existing_code_engine_project_id != null ? var.existing_code_engine_project_id : null
+  code_engine_project_name         = local.code_engine_project_id != null ? null : (var.prefix != null && var.prefix != "") ? "${var.prefix}-code-${local.default_code_engine_project_name}" : local.default_code_engine_project_name
+  code_engine_app_name             = (var.prefix != null && var.prefix != "") ? "${var.prefix}-${local.default_code_engine_app_name}" : local.default_code_engine_app_name
```

I don't think we need an add extra local variable `default_code_engine_app_name` since the input variable itself has a default value. Suggest to go with this:
```suggestion
  code_engine_app_name = (var.prefix != null && var.prefix != "") ? "${var.prefix}-${var.kibana_code_engine_new_app_name}" : var.kibana_code_engine_new_app_name
```